elements/nuxeo-browser/nuxeo-breadcrumb.js

```diff
@@ -247,7 +285,19 @@ import { microTask } from '@polymer/polymer/lib/utils/async.js';
 
     _title(document) {
       if (document) {
-        return document.type === 'Root' ? this.i18n('browse.root') : document.title;
+        return document.type === 'Root' ? this.i18n('browse.root') : `${document.title}`;
+      }
+    }
+
+    _documentUID(document) {
+      if (document) {
+        return `(${document.uid.substring(document.uid.lastIndexOf('-') + 1, document.uid.length)})`.trim();
+      }
+    }
+
+    _isTrashed(document) {
+      if (document) {
+        return document.isTrashed;
```

Comment:
nitpicking: could be made smaller:
```suggestion
      return document && document.isTrashed;
```